src/com/phonebookservice/util/FilterQueryParam.java

```diff
@@ -0,0 +1,5 @@
+package com.phonebookservice.util;
```

Comment:
This isn't utils. This is dispatcher package. 
Why did you descide to add it in Utils ? could you answer please ?

Reply:
cause I thought we will create the dispatcher in another project. but we can add it now in a package.